Loader/Loader/Loader.server.lua

```diff
@@ -9,88 +9,130 @@
 
 
 ----------------------------------------------------------------------------------------
---                                  Adonis Loader                                     --
+--					Adonis Loader					--
 ----------------------------------------------------------------------------------------
---		   	  Epix Incorporated. Not Everything is so Black and White.		   		  --
+--		Epix Incorporated. Not Everything is so Black and White.		--
 ----------------------------------------------------------------------------------------
---	    Edit settings in-game or using the settings module in the Config folder	      --
+--		Edit settings in-game or using the settings module in the Config folder	--
 ----------------------------------------------------------------------------------------
---	                  This is not designed to work in solo mode                       --
+--		This is not designed to work in solo mode.				--
 ----------------------------------------------------------------------------------------
 
-warn(":: Adonis :: Loading...");
+local print = function(...) for i,v in pairs({...}) do warn(":: Adonis ServerLoader :: INFO: "..tostring(v)) end end
+local error = function(...) for i,v in pairs({...}) do warn(":: Adonis ServerLoader :: ERROR: "..tostring(v).."; Traceback:\n"..debug.traceback()) end end
+local warn = function(...) for i,v in pairs({...}) do warn(":: Adonis ServerLoader:: WARN: "..tostring(v)) end end
+local pcall = function(func, ...) local ran, rerror = pcall(func, ...) if not ran then error(rerror) end return ran, rerror end
+local AbortLoad = function(Reason) warn("Adonis aborted loading. Reason: "..tostring(Reason)) if script then script:Destroy() end return false end
```

Comment:
There is no reason to call `tostring` inside of `warn`, as all output functions call it already naturally.
This is why this happens when you print any `Instance`.
```lua
warn(workspace) --> Workspace (equals to workspace.Name)
```

You could improve the `warn` output by utilizing `string.format` to make it more readable.
Here is my suggested approach to making your code more clear and functional.
```suggestion
local function outputMessage(outputType, outputMessage)
    return warn(string.format(":: Adonis ServerLoader :: %s: %s"), outputType, outputMessage)
end

local function pcall(testFunction, ...)
    local success, returnValue = pcall(testFunction, ...)

    if not success then
        return outputMessage("ERROR", returnValue)
    end

    return success, returnValue
end

local function initiateAbort(abortReason)
    outputMessage("ABORTING", string.format("With the reason of %s", abortReason))
    return script:Destroy()
end
```

There is no reason for the `AbortLoad` function to have any return. My suggested change's code is still really unoptimized and a dumb idea overall, but it is definitely better whatever you wrote.

There is no overall improvement in your pull request, it is just randomly changed naming styles for few variables.